### PR TITLE
Fix for trigger function

### PR DESCRIPTION
**Pull request by hkoley**

### CODE REVIEW

File: MC/config/PWGLF/ini/GeneratorLF_Resonances_pp900_trigger.ini [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [GeneratorExternal]
 fileName=${O2DPG_ROOT}/MC/config/PWGLF/pythia8/generator_pythia8_LF.C
-funcName=generateLF("${O2DPG_ROOT}/MC/config/PWGLF/pythia8/generator/resonancelistgun_trig.json", true, 4)
+funcName=generateLFTriggered("${O2DPG_ROOT}/MC/config/PWGLF/pythia8/generator/resonancelistgun_trig.json", true, 4)
```
Comment (BongHwi) [inline MC/config/PWGLF/ini/GeneratorLF_Resonances_pp900_trigger.ini:3]:
```suggestion
funcName=generateLFTriggered("${O2DPG_ROOT}/MC/config/PWGLF/pythia8/generator/resonancelistgun_trig.json", 4)
```

**Review by BongHwi — COMMENTED** [review]
Thanks a lot @hkoley for the update on the triggered production script.

Here are some suggestions.

Have you tried to run this on the grid?

**Comment by BongHwi** [comment]
+async-label async-2022-pp-apass4,async-2022-pp-apass6-2023-PbPb-apass2,async-2023-pbpb-apass3,async-2023-pbpb-apass4

**Comment by BongHwi** [comment]
Please update here after you test the updated script. @hkoley

**Comment by sawenzel** [comment]
@hkoley : I would be appreciated if you could provide a comment  / textual description of what this PR is achieving. From the short commit message "fix trigger" it is not totally clear.

**Comment by sawenzel** [comment]
It appears there was no further activity on this PR for some weeks? Is the PR still needed?

**Pull request closed without merging**